bedrock/newsletter/templates/newsletter/index.html

```diff
@@ -12,7 +12,7 @@
     {# Remove the span tags from the headline next time it changes: #}
     <h1 class="large">{{ _('Read all about it in our <span>newsletter</span>') }}</h1>
     <p>
-      {% trans %}Subscribe to monthly updates and keep current with Mozilla news, including the latest tips and tricks for getting the most out of your Firefox browser. It’s the perfect way for us to keep in touch!{% endtrans %}
+      {% trans %}Subscribe to updates and keep current with Mozilla news. It’s the perfect way for us to keep in touch!{% endtrans %}
```

Comment:
This type of change needs to be behind a l10n_has_tag conditional
http://bedrock.readthedocs.io/en/latest/l10n.html#staging-copy-changes